lib/logger.ts

```diff
@@ -0,0 +1,20 @@
+const noop = () => undefined;
+
+export default class Logger {
```

Comment:
why not use `winston` logger ? No need to reinvent it: https://github.com/standardnotes/syncing-server-js/blob/develop/src/Bootstrap/Container.ts#L78-L88

Reply:
Well, lodash is well known to be bloated in package size but I don't see the point in reinventing the wheel with things like `logger` or `uuid`

Reply:
Payload size is important for this project (as opposed to the server) because this library will be embedded in every editor/component/extension. So if all we need from a logger is something simple it might be better to just write it, then to add a 50KB package that does more than we need it to do. But unsure of the specifics here, just speaking generally.

Reply:
The `logger` is simply used to log to console when `debug` is enabled. For now I think is enough, but this may change in the future.